modules/default/updatenotification/git_helper.js

```diff
@@ -0,0 +1,161 @@
+const util = require("util");
+const exec = util.promisify(require("child_process").exec);
+const fs = require("fs");
+const path = require("path");
+
+class gitHelper {
+	constructor() {
+		this.gitRepos = [];
+		this.baseDir = path.normalize(__dirname + "/../../../");
+		if (process.env.JEST_WORKER_ID === undefined) {
```

Comment:
I dont like having special cases for tests in normal code, but I guess you couldnt find a better way neither? (not sure if other modules have something done like this before)

Reply:
> but I guess you couldnt find a better way neither? 

yes ...